### DISCUSSION TITLE
After Restart Virtual Machine, No Logs in Hunt and Alerts

### DISCUSSION THREAD

**Discussion by anitatata** [Security-Onion-Solutions/securityonion#8516] [Unsupported Versions] [answered]
Hello guys, I have installed Security Onion and it has generated some problems.
So, on 8th August I did restart my virtual machine, but after that there is no log in hunt and alert since that.

Currently I'm still new with Security Onion, so I do not know what to do.
After read the discussion, I follow steps by discussion number #8051.

But, when I try to check Hunt or Alerts, its still not showing anything on 9th August until now.

Please kindly help.
Thanks

**Comment by dougburks** [comment, marked as answer]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.110
- Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, did you install on CentOS 7 or Ubuntu?
- How many nodes do you have?
- What are the hardware specs of each of those nodes?
- How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.

**Reply by anitatata** [reply]
Hello

Thank you and sorry if I'm not clearly explain my problem.

Now my security onion is running well. After search where the error is, the error is Unassigned Shards, so I delete the Unassigned Shards

Thank you
